File: JAVA/Problema1010.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

  
        Scanner scn = new Scanner(System.in);

        int cod1 = scn.nextInt();
        int uni1 = scn.nextInt();
        float pre1 = scn.nextFloat();

        int cod2 = scn.nextInt();
        int uni2 = scn.nextInt();
        float pre2 = scn.nextFloat();

        DecimalFormat df = new DecimalFormat("####.00");

        float res = (uni1 * pre1) + (uni2 * pre2);

        System.out.println("VALOR A PAGAR:" + " " + "R$" + " " + df.format(res));
    }
}

File: C/Problema1010.c
#import <stdio.h>

int main() {
	int cod1, cod2, n1, n2;
	float val1, val2, valpg;

	scanf("%d %d %f", &cod1, &n1, &val1);
	scanf("%d %d %f", &cod2, &n2, &val2);

	valpg = (n1*val1)+(n2*val2);

	printf("VALOR A PAGAR: R$ %.2f\n", valpg);
}
